src/ReflectionFile.php

```diff
@@ -50,6 +50,9 @@ class ReflectionFile
      */
     public function __construct($fileName, $topLevelNodes = null)
     {
+        if (!is_string($fileName)) {
```

Comment:
This was a change from my original commit from July 3rd before I ever issued this PR. I'm slightly confused.